code/DDSCodeTester.cpp

```diff
@@ -565,7 +641,9 @@ void dds_domain_examples()
 
         // Enable statistics DataWriter
         if (statistics_participant->enable_statistics_datawriter(eprosima::fastdds::statistics::GAP_COUNT_TOPIC,
-                eprosima::fastdds::statistics::dds::STATISTICS_DATAWRITER_QOS) != ReturnCode_t::RETCODE_OK);
+                eprosima::fastdds::statistics::dds::STATISTICS_DATAWRITER_QOS) != ReturnCode_t::RETCODE_OK)
+        {
+        }
```

Comment:
This brackets should be removed.

Reply:
Done

Reply:
Have you pushed your local changes? It seems that these brackets are still appearing in the PR.

Reply:
Up, forget it. Pushed now.